test/fail_compilation/imports/a18243.d

```diff
@@ -0,0 +1,5 @@
+module a18243;
+
+import std.math : isNaN;
```

Comment:
Selective imports creating`alias` in the current scope is really an annoyance, and violates the principle of least surprise. I wish we could change that.

Reply:
That is a small project on its own which will propagate cascade changes throughout a large number of components.

Reply:
Although I agree that the current state is hackish and messy.

Reply:
Last time I checked with Walter, it was by design. Maybe that changed since. If you have any way to find out, it would be appreciated.